src/commands/fail_if_dirty.yml

```diff
@@ -41,7 +41,7 @@ steps:
 
         FILES_MODIFIED=""
         setcommit () {
-          FILES_MODIFIED=$(git status -s | grep -i <<# parameters.invert-pattern >> -v <</ parameters.invert-pattern >> -E '<< parameters.pattern >>')
+          FILES_MODIFIED=$(git status -s | grep <<# parameters.invert-pattern >> -v <</ parameters.invert-pattern >> ${SWISSKNIFE_GREP_FLAGS:--i -E} '<< parameters.pattern >>')
```

Comment:
we want the `-i` here irrespective  of invert right?

Reply:
The `-i` is part of the default `SWISSKNIFE_GREP_FLAGS` value. So it should expand to `-i -E <patt>` or `-v -i -E <patt>` if invert-pattern is specified. Setting `SWISSKNIFE_GREP_FLAGS` will override `-i -E` in both cases. Is that correct?